src/scss/includes/panels/poi_panel.scss

```diff
@@ -78,6 +78,11 @@ $BLOCK_ICON_FONT_SIZE: 16px;
   .poi_panel__action__direction {
     flex-grow: 1;
   }
+
+  .minimized:not(.panel--holding) & {
+    height: 0;
```

Comment:
Just to be sure, I'm curious about this choice. Didn't it work with simply `display: none/block`?

Reply:
It could have been a display toggle indeed, but as stated in another discussion, using the height could be part of the solution to reintroduce transitions between default abd minimized. (didn't have time to test it yet)

Reply:
I think this is unrelated. You'll need to get the real height of the whole panel at some points. Having just a fixed height for only a subpart of it doesn't give any hint about the height of the rest. Using `auto` is still mandatory if we want to adapt to possibly very long titles and the presence/absence of some data (reviews, etc.).
So if this was the only reason to use `height` instead of `display`, you can simplify it by using the latter. Let's not introduce dependencies between separate tickets.
Even if it turns we need height in the other fix, it's pretty straightforward to change.